core/src/main/res/layout/main_activity_content.xml

```diff
@@ -20,14 +20,16 @@
 
       <include layout="@layout/layout_toolbar" />
 
-      <org.kiwix.kiwixmobile.core.main.AnimatedProgressBar
+      <androidx.core.widget.ContentLoadingProgressBar
```

Comment:
set height to 2dp, the line still looks a bit fat, this also means you can just delete and inline the NoPadding style here, removing the min max properties. Creating a style for this UI element is unnecessary because there is only 1 of it

This still can not be previewed in the layout preview pane so add `tools:visiblity="visible"`

Delete `app:backgroundColor="?colorOnSurface"` as that seems like a pointless relic

And last but not least `ctrl+alt+l` to reformat this file, android xml attributes have an innate order.

Reply:
Sure👍